### ISSUE TITLE
Default operator behaviour '=' instead of specified in field.operators

### ISSUE THREAD

**Issue by protzi** [react-querybuilder/react-querybuilder#162]
Hi,
If you will specify a custom operator, the default operator value will be '=' instead of '===' and it will still be the same even if you pick manually after initialization.

```
    {
        name: 'status',
        label: 'Status',
        operators: [ {name: '===', label: 'is'} ],
        valueEditorType: 'radio',
        values: [
            { name: 'active', label: 'Active' },
            { name: 'paused', label: 'Paused' }
        ]
    }
```

![image](https://user-images.githubusercontent.com/17887536/96972914-d5332380-151f-11eb-8ab7-093e62f8560e.png)

**UPDATE: SOLVED** 

```
    getOperators works as expected

    getOperators={(field) => {
      return fieldOperatorMap[field];
    }}
```

**Comment by jakeboone02** [comment]
We still need to fix the issue with the `operators` though, so I'm reopening this issue.

**Comment by jakeboone02** [comment]
This is fixed with commit 5ad9da5d; released in version [3.8.1](https://github.com/sapientglobalmarkets/react-querybuilder/releases/tag/v3.8.1).
